src/routes/Home.js

```diff
@@ -1,11 +1,44 @@
 import React from 'react';
 
 import OrganizationsList from '../components/OrganizationsList';
+import Input from '../ui-kit/components/Input';
+import Button from '../ui-kit/components/Button';
+
+import styled from 'styled-components';
+
+const StyledForm = styled.form`
```

Comment:
Could we write these components without the 'styled' prefix? Just 'Form' (capital letter) or 'Label'

Reply:
I think we could